src/build/rewrite-webcomponents-loader.ts

```diff
@@ -49,25 +73,78 @@ export class UseES5WebcomponentsLoader extends stream.Transform {
         stream.on('error', reject);
       });
     }
-    if (!/webcomponents-loader\.js/.test(contents)) {
+    if (!webcomponentsLoaderRegex.test(contents)) {
       callback(null, file);
       return;
     }
     const parsed = parse5.parse(contents);
-    const script = dom5.nodeWalk(parsed, scriptIncludeWebcomponentsLoader);
+    const script = dom5.nodeWalk(parsed, webcomponentsLoaderMatcher);
     if (!script) {
       callback(null, file);
       return;
     }
 
-    const scriptPath = dom5.getAttribute(script, 'src')!;
-    const scriptUrl = url.resolve(scriptPath, 'custom-elements-es5-adapter.js');
-    const es5AdapterScript =
-        parse5.parseFragment(`<script src="${scriptUrl}"></script>`);
-    dom5.insertBefore(script.parentNode!, script, es5AdapterScript);
-
+    // Collect important dom references & create fragments for injection
     const correctedFile = file.clone();
+    const headElement = dom5.query(parsed, headMatcher)!;
+    const bodyElement = dom5.query(parsed, bodyMatcher)!;
+    const loaderScriptUrl = dom5.getAttribute(script, 'src')!;
+    const adapterScriptUrl =
+        url.resolve(loaderScriptUrl, 'custom-elements-es5-adapter.js');
+    const loaderMovedComment = dom5.constructors.comment(
+        ' AUTO-GENERATED BY JS-COMPILE: "webcomponents-loader.js" and ' +
```

Comment:
JS-COMPILE is isn't super clear. How about:

` AUTO-GENERATED BY polymer-build: `